docs/working-with-groups.md

```diff
@@ -98,6 +100,12 @@ for (Group actGroup : searchResult.Resources()) {
 //...
 ```
 
+It is possible to filter the attributes of the returned groups by supplying the attributes as parameters to the merhod:
+
+```java
+SCIMSearchResult<Group> searchResult = oConnector.getGroup(accessToken, "displayName", "meta.created");
```

Comment:
The example does not fit the actual method name and signature: `public SCIMSearchResult<Group> searchGroups(Query query, AccessToken accessToken)`